tools/mason/MasonUtils.chpl

```diff
@@ -118,6 +121,50 @@ proc runWithStatus(command, show=true): int {
   }
 }
 
+
+/* uses spawnshell and the prefix to setup Spack before
+   calling the spack command. This also returns the stdout
+   of the spack call.*/
+proc getSpackResult(cmd, quiet=false) : string throws {
+  var ret : string;
+  try {
+    
+    var prefix = "export SPACK_ROOT=/Users/spartee/.mason/spack" +
+    " && export PATH=$SPACK_ROOT/bin:$PATH" +
+    " && source $SPACK_ROOT/share/spack/setup-env.sh && ";
+    var splitCmd = prefix + cmd;
+    var process = spawnshell(splitCmd, stdout=PIPE);
```

Comment:
Can we add `stderr=PIPE` here (and other spack calls) to see if we can prevent spack errors from reaching the user?

Reply:
Good suggestion. Will try this.

Reply:
this worked was able to sed the output and replace with ``mason external``. Given that our commands mirror that of Spack's commands, the errors now represent accurate errors for Mason as well.